tasks/js-oop.md

```diff
@@ -28,22 +28,10 @@ When complete do the following:
 1. You did a lot already! If you honestly finished all the previous steps then go ahead
    and share it with others –
    post a message in [course channel][chat]:
-   `Object Oriented JS — #done` and add the link to your repo. **This step is important, as it helps mentors to track your progress!**
+   `Object Oriented JS — #done`(or `Object Oriented JS — #p2p-pr-done` if you are p2p course student)  and add the link to your repo. **This step is important, as it helps mentors to track your progress!**
 1. You will require code review for a coding subtask from (1)
-   (arcade game **Frogger**):
-   * publish your task code base for review following instructions
-     in [FE 2019 course homeworks repo](https://github.com/kottans/frontend-2019-homeworks)
-   * post a link to your subtask PR in the
-     [FE Questionarium chat](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw)
-     and ask mentors for a code review
-   * once the code review phase successfully finished
-     and you have an approval from one of the mentors
-     add to your `kottans-frontend/README.md`
-     a link to subtask code base and
-     a note saying `Reviewed and approved by @<mentor>`
-     (use mentor's github username)
-   * list your reflections on this subtask
-     (_what was new to you_, _what surprised you_, _what you intend to use in future_)
+   - If you are **p2p course** student follow next [instructions](https://github.com/Oodmincheg/frontend-2019-p2p/blob/master/CONTRIBUTING.md)
```

Comment:
it depends. Instructions take place in CONTRIBUTING.md. Sot it can be confusing for student follow the link and see general description about p2p-homework repo. This wasn't a problem, cause in kottans-homework repo instructions take place in README.md.

Reply:
I propose don't do move instruction into README.md in p2p-homework too. I rememeber your thoughts about conributing. but in will be a little mess for students who already have started the course. and it isn't a contribunting in the strictest sense of the word.

Reply:
Do you mean all instructions in `readme.md`? So we will not have `contributing.md`?

LGTM.

Reply:
Yes. I mean that p2p-homework will not have <contributing.md> like kottans-homework. and links will be forward to the repos(readme.me) in both variants. I can close PR's and do some changes?

Reply:
@Oodmincheg Let's try having contribution rules in `frontend-2019-p2p/README.md`. Anyway it will contain link to `code-review-guidelines.md`

Reply:
IMO, we can drop this one `code-review-guidelines.md` and merge content into `readme.md`.

Reply:
Yeah, I like an idea. We can split it after we will have what to split.